README.md

```diff
@@ -116,6 +116,21 @@ tango info
 
 to check your installation.
 
+### Docker image
+
+You can build a Docker image suitable for tango projects by using [the official Dockerfile](https://github.com/allenai/tango/blob/main/Dockerfile) as a starting point for your own Dockerfile, or you can simply use one of our [prebuilt images](https://github.com/allenai/tango/pkgs/container/tango) as a base image in your Dockerfile. For example:
+
+```Dockerfile
+FROM ghcr.io/allenai/tango:cuda11.5.1
```

Comment:
Do you want to include a version of tango in the image id?

Reply:
This better? e208bcb

Reply:
I didn't want to hard-code a version since that would quickly go out-of-date.

Reply:
We won't have a versioned tag (like `tango:0.4.1-cuda11.5.1`) until the next release.